### PR TITLE
Junit

### PR DESCRIPTION
**Pull request by melnikovmaks**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+srlenium.iml
+build/
```
Comment (Kirill-Simonovich) [inline .gitignore:3]: read about gitignore and add more points to this list, you have example in GitHub 'Gsuite auto-test '

File: src/test/java/BaseTest.java [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-import static com.codeborne.selenide.Selenide.closeWebDriver;
 
 abstract public class BaseTest {
 
     @BeforeAll
-    public static void setAll() {
+    public static void setUp() {
         Configuration.browser = "chrome";
-        Configuration.browserVersion = "92.1";
+        Configuration.browserVersion = "97.0";
```
Comment (Kirill-Simonovich) [inline src/test/java/BaseTest.java]: share your Chrome browserVersion

File: src/test/java/TestClass.java [hunk 3]
```diff
@@ -6,48 +6,56 @@
...
-import java.util.Random;
 
 import static com.codeborne.selenide.Selenide.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class TestClass extends BaseTest {
-    public static String baseUrl = "http://automationpractice.com/";
+    static final String BASE_URL = "http://automationpractice.com/";
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: find a better place in code for this variable

File: src/test/java/TestClass.java [hunk 4]
```diff
@@ -6,48 +6,56 @@
...
 
     @Disabled("It is not useful test")
     @Test
-    public void test() {
-        open(baseUrl);
-        $$(".item-img").should(CollectionCondition.size(7)).get(4).scrollTo();
+    public void scrollTo() {
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: Example "checkScroll", 
or are you testing the Selenide scrollTo () method?

File: src/test/java/TestClass.java [hunk 5]
```diff
@@ -6,48 +6,56 @@
...
+        open(BASE_URL);
+        ElementsCollection images = elements(".item-img");
+        images.should(CollectionCondition.size(7)).get(4).scrollTo();
     }
 
     @RepeatedTest(2)
     @DisplayName("Choice of color and size")
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: Check choosing of color and size in Faded Short Sleeve T-shirts page

File: src/test/java/TestClass.java (hunk 4 above)
Comment (Kirill-Simonovich) [reply src/test/java/TestClass.java]: rename everywhere

File: src/test/java/TestClass.java (hunk 5 above)
Comment (Kirill-Simonovich) [reply src/test/java/TestClass.java]: rename everywhere

File: src/test/java/TestClass.java [hunk 6]
```diff
@@ -6,48 +6,56 @@
...
+        assertTrue(currentUrl.endsWith("color-orange"));
+        SelenideElement blueColor = element(".color_pick:not(#color_13)");
+        blueColor.click();
         currentUrl = WebDriverRunner.getWebDriver().getCurrentUrl();
-        Assertions.assertTrue(currentUrl.contains("color-blue"));
-        SelenideElement element = $("#group_1");
-        if (element.is(Condition.name("group_1"))) {
-            element.click();
+        assertTrue(currentUrl.contains("color-blue"));
+        SelenideElement sizeBar = element("#group_1");
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: sizeSelector better

File: src/test/java/TestClass.java [hunk 7]
```diff
@@ -6,48 +6,56 @@
...
+        SelenideElement orangeColor = element("#color_13");
+        orangeColor.shouldNot(Condition.image).click();
+        assertEquals(orangeColor.getLocation(), new Point(1136, 640));//1019
+        assertEquals(orangeColor.getSize(), new Dimension(22, 22));
+        assertEquals(orangeColor.getCssValue("background-color"), "rgba(243, 156, 17, 1)");
         String currentUrl = WebDriverRunner.getWebDriver().getCurrentUrl();
-        Assertions.assertTrue(currentUrl.endsWith("color-orange"));
-        $(".color_pick:not(#color_13)").click();
+        assertTrue(currentUrl.endsWith("color-orange"));
+        SelenideElement blueColor = element(".color_pick:not(#color_13)");
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: blueColorPick

File: src/test/java/TestClass.java [hunk 8]
```diff
@@ -6,48 +6,56 @@
...
-        Assertions.assertEquals($("#color_13").getLocation(), new Point(1136, 640));//1019
-        Assertions.assertEquals($("#color_13").getSize(), new Dimension(22, 22));
-        Assertions.assertEquals($("#color_13").getCssValue("background-color"), "rgba(243, 156, 17, 1)");
+    public void choiceOfColorAndSize() {
+        open(BASE_URL);
+        SelenideElement fadedClothes = element("[alt^='Faded']");
+        fadedClothes.shouldBe(Condition.appear, Duration.ofSeconds(20)).hover();
+        ElementsCollection buttonsMore = elements(".lnk_view > span");
+        buttonsMore.should(CollectionCondition.size(14)).get(0).click();
+        SelenideElement orangeColor = element("#color_13");
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: orangeColorPick

File: src/test/java/TestClass.java [hunk 9]
```diff
@@ -6,48 +6,56 @@
...
-        open(baseUrl);
-        $("[alt^='Faded']").shouldBe(Condition.appear, Duration.ofSeconds(20)).hover();
-        $$(".lnk_view > span").should(CollectionCondition.size(14)).get(0).click();
-        $("#color_13").shouldNot(Condition.image).click();
-        Assertions.assertEquals($("#color_13").getLocation(), new Point(1136, 640));//1019
-        Assertions.assertEquals($("#color_13").getSize(), new Dimension(22, 22));
-        Assertions.assertEquals($("#color_13").getCssValue("background-color"), "rgba(243, 156, 17, 1)");
+    public void choiceOfColorAndSize() {
+        open(BASE_URL);
+        SelenideElement fadedClothes = element("[alt^='Faded']");
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: fadedTShirt

File: src/test/java/TestClass.java (hunk 7 above)
Comment (Kirill-Simonovich) [reply src/test/java/TestClass.java]: check all naming, the blue color is not clear what it indicates

File: src/test/java/TestClass.java [hunk 10]
```diff
@@ -58,80 +66,89 @@ public void test1() {
...
     })
-    public void test2(String mail, String password, String message) {
-        open(baseUrl);
-        $(".login").shouldBe(Condition.visible, Duration.ofSeconds(20)).click();
-        $("#email").shouldBe(Condition.empty).setValue(mail);
-        $("#passwd").shouldBe(Condition.empty).setValue(password).pressEnter();
-        Assertions.assertEquals(message, $(".account").shouldBe(Condition.exist,
+    public void loginOnSite(String mail, String password, String message) {
+        open(BASE_URL);
+        SelenideElement login = element(".login");
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java]: loginField

**Comment by Kirill-Simonovich** [comment]
Delete unnecessary code from main folder

**Review by Kirill-Simonovich — CHANGES_REQUESTED** [review]

**Comment by Kirill-Simonovich** [comment]
tests don't work
image

File: src/test/java/BaseTest.java (hunk 2 above)
Comment (melnikovmaks) [reply src/test/java/BaseTest.java]: deleted this configuration

File: src/test/java/TestClass.java (hunk 3 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: moved BASE_URL to the right place

File: src/test/java/TestClass.java (hunk 4 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename

File: src/test/java/TestClass.java (hunk 5 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename all DisplayName

File: src/test/java/TestClass.java (hunk 6 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename

File: src/test/java/TestClass.java (hunk 7 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename all colors

File: src/test/java/TestClass.java (hunk 8 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename all colors

File: src/test/java/TestClass.java (hunk 9 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename all clothes

File: src/test/java/TestClass.java (hunk 10 above)
Comment (melnikovmaks) [reply src/test/java/TestClass.java]: rename

**Comment by Kirill-Simonovich** [comment]
Rename variables, delete empty string, add workspace.xml to gitignore

File: src/test/java/TestClass.java [hunk 11]
```diff
@@ -6,132 +6,146 @@
...
-    public static String baseUrl = "http://automationpractice.com/";
 
     @Disabled("It is not useful test")
     @Test
-    public void test() {
-        open(baseUrl);
-        $$(".item-img").should(CollectionCondition.size(7)).get(4).scrollTo();
+    public void checkScroll() {
+        open("/");
+        ElementsCollection images = elements(".item-img");
```
Comment (Kirill-Simonovich) [inline src/test/java/TestClass.java:21]: nice name

**Review by Kirill-Simonovich — APPROVED** [review]

**Pull request merged**